### PR TITLE
Blog App: processing data in models

### PR DESCRIPTION
**Pull request by ibr5500**
## In this [PR] I have:

> - Create posts counter method.
> - Create comments counter method.
> - Create likes counter method.
> - Create a method for the most 3 recent posts
> - Create a method for the most 5 recent comments
> - Create users using rails c.
> - Create posts using rails c.
> - Create comments using rails c.
> - Create likes using rails c.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,3 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts, :comments, :likes
+  has_many :posts
+  has_many :comments
+  has_many :likes
```
Comment (prolajumokeoni) [inline app/models/user.rb]:
```suggestion
  has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

- [x] Kindly make sure that user models have a foreign key with author_id for posts, comments and likes 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @ibr5500, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @prolajumokeoni for the code review.

**Review by elmar8287 — COMMENTED** [review]
Hi, @ibr5500.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- All suggestions are implemented 👍 

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @elmar8287 for the code review
.

**Pull request merged**